src/frontend/src/lib/utils/transactions.utils.ts

```diff
@@ -40,8 +58,24 @@ export const mapAllTransactionsUi = ({
 		}
 
 		if (isNetworkIdEthereum(networkId)) {
-			// TODO: Implement Ethereum transactions
-			return acc;
+			// TODO: remove Sepolia transactions when the feature is complete; for now we use it for testing
+			const isSepoliaNetwork = isNetworkIdSepolia(networkId);
+
+			return [
+				...acc,
+				...($ethTransactions[tokenId] ?? []).map((transaction) => ({
+					...mapEthTransactionUi({
+						transaction,
+						ckMinterInfoAddresses: toCkMinterInfoAddresses({
```

Comment:
I'm not sure how "heavy" is but, maybe interesting to compute it only once and not within the reduce?

Reply:
it varies from token to token, but you mean to get the one for mainnet and the one for testnet, and then assign each one, depending on the token? so that we call the function `toCkMinterInfoAddresses` only once?

Reply:
Even if `toCkMinterInfoAddresses` appears to actually be trivial, I would maybe still do so, WDYT?

Reply:
yes agreed, have a look at last commit, I had to use mutable variables.... not really a fan of it

Obviously i can just calculate it upfront, but it seemed correct to calculate it only if and when we have the need

Reply:
or do you prefer immutable?